packages/cli/core/src/lib/atomic/createAtomicProject.ts

```diff
@@ -86,16 +87,20 @@ interface CreateLibOptions {
   projectName: string;
 }
 
-export function createAtomicLib(options: CreateLibOptions) {
+export async function createAtomicLib(options: CreateLibOptions) {
   const projectDirectory = resolve(options.projectName);
   mkdirSync(projectDirectory);
-  const cliArgs = [
-    'init',
-    `${transformPackageNameToNpmInitializer(
-      atomicLibInitializerPackage
-    )}@${getPackageVersion(atomicAppInitializerPackage)}`,
-  ];
-  return spawnProcess(appendCmdIfWindows`npm`, cliArgs, {
+  const initializer = `${atomicLibInitializerPackage}@${getPackageVersion(
+    atomicAppInitializerPackage
+  )}`;
+
+  const forkedProcess = npf(initializer, [], {
+    stdio: 'inherit',
     cwd: projectDirectory,
   });
+  try {
+    await handleForkedProcess(forkedProcess);
+  } catch (error) {
+    throw error;
```

Comment:
- [ ] @louis-bompart : Manual check up, run `coveo atomic:init` in a folder with a rogue file.

Reply:
- [ ] use SubprocessError